swing/src/net/sf/openrocket/gui/dialogs/ComponentAnalysisDialog.java

```diff
@@ -165,17 +163,20 @@ public void stateChanged(ChangeEvent e) {
 
 		////Angle of attack:
 		panel.add(new JLabel(trans.get("componentanalysisdlg.lbl.angleofattack")), "width 120lp!");
-		panel.add(new UnitSelector(aoa, true), "width 50lp!");
+		panel.add(new EditableSpinner(aoa.getSpinnerModel()), "growx");
+		panel.add(new UnitSelector(aoa), "width 50lp!");
 		panel.add(new BasicSlider(aoa.getSliderModel(0, Math.PI)), "growx, wrap");
 
 		//// Mach number:
 		panel.add(new JLabel(trans.get("componentanalysisdlg.lbl.machnumber")), "width 120lp!");
+		panel.add(new EditableSpinner(mach.getSpinnerModel()), "width 120lp!");
 		panel.add(new UnitSelector(mach, true), "width 50lp!");
```

Comment:
Remove the `true` argument, we don't want the mach value to be displayed in the unit selector.